Pipfile.lock

```diff
@@ -33,10 +33,10 @@
         },
         "certifi": {
             "hashes": [
-                "sha256:47f9c83ef4c0c621eaef743f133f09fa8a74a9b75f037e8624f83bd1b6626cb7",
```

Comment:
I run ```pipenv update``` and Pipfile.lock got changed. Should I put the previous back?

Reply:
Updates are done by our bots. If you did not introduce any new package, you don't need to commit Pipfile.lock - you can let bots do that (and keep PRs atomic).